### DISCUSSION TITLE
Notes - 13th Feb 2024

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#9] [General]
## Leverage

@HumphreyYang discussed the idea of leverage and presented a very interesting notebook on the topic. It is available in `resources` in this repo. 

## Review of Notebook (GDP Correlations Update)

TBD

GDP Compositions

- GDP composition plots (as a share of GDP)
- GDP composition across countries (if data is available) -- for example share of Investment of GDP across various levels of development. 

**Access to World Bank Data:**

- My recommendation for World Bank data access is [wbgapi](https://blogs.worldbank.org/opendata/introducing-wbgapi-new-python-package-accessing-world-bank-data) a good package to use for access to API data from `python`

## Review of Business Cycles Lecture [business_cycle]

- @mmcky to work on the theme update for `Show code` rather than `Show code cell output` and `Hide code` rather that `Hide code cell output`
- Is having the US and the UK redundant? 
- Possible connect UK with the longer time series for the UK in `long run growth` to look at fluctuations over a longer time horizon over the Industrial Revolution period. 
- **ACTION:**  Update this lecture to the latest data available

### Section: Unemployment

- How is unemployment related to business cycles? Clarify the linkage between GDP and unemployment. Business cycles are generally a vague term -- however measuring of recessions is in terms of GDP growth. How are recessions defined? 
- Good use of drop downs for the code cells. 
- Does unemployment recover quicker after a recession compared to Europe (due to labour market frictions) -- is there a difference in speed of rise and/or fall. 

### Section: Leading indicators and correlated factors

**Consumer Sentiment**

- what is consumption?
- what is the consumer sentiment index measuring? 
- What is cpi? 

Figure 3.10 -- the focus here is on consumer sentiment so perhaps we focus on consumer sentiment, and then look at drivers of consumer sentiment in relation to CPI. So we could split this graph into two steps. 

**Production:**

- update variable `consumer_sentiment` (@HumphreyYang)
- How is production linked to business cycles

**Credit Levels:** 

- what is it and how would it link to business cycles
- We could also show the US? 

@Jiarui-ZH to lead review of `inequality` lecture (Lecture #4)

**Comment by HumphreyYang** [comment]
Here is a lecture using World Bank API:
https://intro.quantecon.org/business_cycle.html

**Reply by mbek0605** [reply]
On top of Humphrey's notes, I will do the following:

1. Cross-country comparison: poor countries vs. rich countries -> how much of their GDP are they investing/consuming (data collection??)
2. Find another way to visualize the components of GDP (trend and aggregate nature).
3. (For Jiaru too) Include more years before 1880 to highlight the causal nature

**Comment by HumphreyYang** [comment]
Notes on Business Cycle by @pgrosser1 (Thanks Phoebe!):

1. Add motivations on what is the business cycle and connect to the long-run growth lecture.
2. The country-by-country plots are perhaps redundant.
3. Update Argentina data (or all data).
4. Explain why unemployment is an important measure of business cycles (add recent strong employment data into discussion)
5. Add European unemployment data to show the response of the labor market after the recession.
6. Discuss concepts used as leading indicators and how they link to the business cycle.
7. Break up the plot on consumer confidence and CPI.

**Comment by mmcky** [comment]
@QuantEcon/reading-group-2024 I have set up actions that relate to these notes

- [ ] https://github.com/QuantEcon/lecture-python-intro/issues/364
- [ ] https://github.com/QuantEcon/lecture-python-intro/issues/365
- [ ] https://github.com/QuantEcon/lecture-python-intro/issues/366
- [ ] https://github.com/QuantEcon/lecture-python-intro/issues/367
- [ ] https://github.com/QuantEcon/lecture-python-intro/issues/368

@HumphreyYang if you could please review to make sure I haven't missed anything. Also please feel free to edit as you see fit to improve clarity or add further details.
